### DISCUSSION TITLE
Customizables Keybindings

### DISCUSSION THREAD

**Discussion by ByronJim** [seerge/g-helper#1867] [Ideas]
Hi, first thanks for the software
Can you add customizable options to m-buttons, actually  Ctrl+m-buttons or Shift+m-buttons are used for  Screen brightness and Backlight brightness and it can be used for example in media control play/pause, prev next.

**Comment by seerge** [comment]
@ByronJim hello, I don't really want to overload app with lots of options and settings :) But you can try to make bindings by yourself, using Sharpkeys for example.
